<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InterfaceTab</class>
 <widget class="QWidget" name="InterfaceTab">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>953</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget{
background-color:black;
color: gray;
selection-background-color:#186370;
selection-color: #c5ba94;
}


QMenu{
font-size:17px;
font-weight: bold;
width: 150px;
left: 35px;
color: white;
background-color:#21a0b7;
}

QToolButton {
border-radius: 10px;
color:white;
border: 1px solid #0f4b50;
background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
stop: 0 #21a0b7, stop: 1.0 #186370);
}

QToolButton:pressed{
border-radius: 10px;
color:white;
border: 1px solid #161d1e;
background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
stop: 0 #0b393d, stop: 1.0 #105256);
}

QToolButton:disabled{
border-radius: 10px;
color:#6F9499;
border: 1px solid #161d1e;
background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
stop: 0 #283233, stop: 1.0 #283233);
}

QLineEdit {
border-radius: 10px;
border: 2px solid qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
stop: 0 #21a0b7, stop: 1.0 #186370);
}


QRadioButton::indicator::unchecked {
    image: url(://data/icons/btn_radiobutton_unchecked.png);
}

QRadioButton::indicator:unchecked:hover {
    image: url(://data/icons/btn_radiobutton_checked_active.png);
}

</string>
  </property>
  <widget class="QRadioButton" name="radioButtonManual">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>20</y>
     <width>241</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Ręcznie</string>
   </property>
   <property name="iconSize">
    <size>
     <width>64</width>
     <height>64</height>
    </size>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="autoExclusive">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditMask">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>278</y>
     <width>410</width>
     <height>64</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButtonDHCP">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>20</y>
     <width>241</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>DHCP</string>
   </property>
   <property name="iconSize">
    <size>
     <width>64</width>
     <height>64</height>
    </size>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="autoExclusive">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditIp">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>150</y>
     <width>410</width>
     <height>64</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>150</y>
     <width>231</width>
     <height>64</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Adres IP:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>278</y>
     <width>231</width>
     <height>64</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Maska podsieci:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditGateway">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>400</y>
     <width>410</width>
     <height>64</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>400</y>
     <width>231</width>
     <height>64</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Brama domyślna:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
